WolvenKit.App/ViewModels/Editor/Tools/ImportExportViewModel.cs

```diff
@@ -646,16 +697,162 @@ private async Task ExecuteProcessSelected()
                 }
                 await ImportWavs(wavs);
             }
-            else
+            if (IsExportsSelected)
             {
                 foreach (var item in ExportableItems.Where(_ => _.IsChecked))
                 {
                     await ExportSingle(item);
                 }
             }
+            if (IsConvertsSelected)
+            {
+                foreach (var item in ConvertableItems.Where(_ => _.IsChecked))
+                {
+                    await ConvertSingle(item);
```

Comment:
Call like this to avoid blocking: await Task.Run(() => ConvertSingleAsync(item));

Reply:
You can just do:
```csharp
_ = ConvertSingleAsync(item)
```